OptimizelySDK/OptimizelySDK.csproj

```diff
@@ -101,8 +101,9 @@
     <Compile Include="Odp\Entity\OdpEvent.cs" />
     <Compile Include="Odp\Entity\Response.cs" />
     <Compile Include="Odp\Enums.cs" />
+    <Compile Include="Odp\IOdpManager.cs" />
     <Compile Include="Odp\IOdpSegmentManager.cs" />
-    <Compile Include="Odp\OdpConfig.cs" />
```

Comment:
This was already declared on line 112/113 below.  Added  `OdpManager.cs` in its place.